client/src/pages/tasks/Show.js

```diff
@@ -100,17 +100,40 @@ const GQL_GET_TASK = gql`
         }
       }
       customFields
-      ${GRAPHQL_NOTES_FIELDS}
+      ${GRAPHQL_NOTES_FIELDS},
+      publishedReports: reports(query: {
```

Comment:
Be aware that this is now potentially a very expensive operation, requesting *all* published reports for a task, and *all*  published reports for *all* sub-tasks.

Reply:
indeed. Could we somehow drop getting the report list, but instead get somehow the report status through the notes? @gjvoosten could you suggest something?

Reply:
At least filter the published reports (both here and on sub-tasks) on engagement date (or release date) within the assessment period.

Reply:
Will be picked up as part of #2954